src/features/cell-renderers/userCellRenderer.js

```diff
@@ -0,0 +1,13 @@
+// @flow
+import * as React from 'react';
+
```

Comment:
The pattern to organize the import seems little weird to me. Is there a pattern that BUIE follows? If not, I would recommend doing the following
```
package import
[new line]
full path import
[new line]
dot dot import
[new line]
type import
```

Reply:
BUIE's pattern is having no [new line]

Reply:
Organized some according to your advice. Thanks!